v2/pkg/operators/common/dsl/dsl_test.go

```diff
@@ -314,7 +314,7 @@ func TestDateTimeDslExpressions(t *testing.T) {
 			`date_time("%Y-%M-%D", 1642032000)`:          time.Date(2022, 01, 13, 0, 0, 0, 0, time.UTC).Local().Format("2006-01-02"),
 			`date_time("%H-%m")`:                         fmt.Sprintf("%02d-%02d", now.Hour(), now.Minute()),
 			`date_time("02-01-2006 15:04", unix_time())`: now.Format("02-01-2006 15:04"),
-			`date_time("02-01-2006 15:04", 1642032000)`:  time.Date(2022, 01, 13, 0, 0, 0, 0, time.UTC).Local().Format("02-01-2006 15:00"),
+			`date_time("02-01-2006 15:04", 1642032000)`:  time.Date(2022, 01, 13, 0, 0, 0, 0, time.UTC).Local().Format("02-01-2006 15:04"),
```

Comment:
This change is not required, it is just a representation, not actual time.

Reply:
Reverted it.